### PR TITLE
Update makefile to add a comment about usage

### PR DESCRIPTION
**Pull request by cooljeanius**
Just a minor commentary addition to address the issue I experienced in testing #295

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Note: this makefile expects you to have run `npm install` and `npm run build`
+# first before using it.
+
 _ := $(shell rm -rf build && npm run build)
```
Comment (rougetimelord) [inline makefile:4]:
```suggestion
_ := $(shell rm -rf build && npm install && npm run build)
```

Wouldn't this work to run all of the necessary commands beforehand?

Comment (cooljeanius) [reply makefile:4]: I mean, I'm kinda wary about adding further stuff to that command...

Comment (cooljeanius) [reply makefile:4]: > I mean, I'm kinda wary about adding further stuff to that command...

...but yeah, I agree that we should check with @kheina

**Comment by cooljeanius** [comment]
Re: [f809e47](https://github.com/kheina-com/Blue-Blocker/pull/308/commits/f809e4722219eb977fd4c0d4155b2cdf71120a77): ok, that looks fine... I can't approve it since it's still part of my own PR, though...

**Comment by rougetimelord** [comment]
@kheina pweaseeeeee

**Comment by cooljeanius** [comment]
> @kheina pweaseeeeee

We really ought to find another person to give reviewer privileges to....

**Review by rougetimelord — APPROVED** [review]

**Comment by rougetimelord** [comment]
There we go

**Pull request merged**